src/app/features/ahbs/components/pruefi-input/pruefi-input.component.ts

```diff
@@ -67,4 +67,14 @@ export class PruefiInputComponent implements ControlValueAccessor {
       this.control.enable();
     }
   }
+
+  onInputChange(event: Event): void {
+    const input = event.target as HTMLInputElement;
+    let value = input.value.replace(/[^0-9]/g, '');
```

Comment:
```suggestion
    let value = input.value.replace(/\D/g, '');
```
`\D` ist alles außer `\d` 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape